src/backend/distributed/sql/udfs/citus_tables/latest.sql

```diff
@@ -18,3 +18,6 @@ WHERE
   partkey IS NOT NULL OR repmodel = 't'
 ORDER BY
   logicalrelid::text;
+
+ALTER VIEW public.citus_tables SET SCHEMA pg_catalog;
```

Comment:
Note that putting this in the public schema was intentional to make users aware that there are Citus tables and that they don't have any yet. We could consider putting it in pg_catalog only when the public schema does not exist.

Reply:
I updated the migration script to check for the existence of public schema. If it exists, we create the view in it. Otherwise we create citus.citus_tables and then move it to pg_catalog.

On downgrades, we drop citus_tables views in any of the two schema in case they exist there.